tests/testthat/tests_estCrossVal.R

```diff
@@ -0,0 +1,39 @@
+context("testing estCrossVal")
+
+test_that("estCrossVal adds correct, new columns", {
+  
+  # Choptank_eList has model results already, but the model 
+  # hasn't been run since it was originally saved.
+  eList <- Choptank_eList
+  info_stale <- getInfo(eList)
+  daily_stale <- getDaily(eList)
+  sample_stale <- getSample(eList)
+  
+  info_orig <- info_stale[, 1:(which(names(info_stale) == "bottomLogQ") - 1)]
+  daily_orig <- daily_stale[, 1:(which(names(daily_stale) == "yHat") - 1)]
+  sample_orig <- sample_stale[, 1:(which(names(sample_stale) == "yHat") - 1)]
+  surfaces_orig <- NA
+  eList_orig <- mergeReport(info_orig, daily_orig, sample_orig, surfaces_orig)
```

Comment:
if there are going to be more than 3 of these code chunks (lines 7-16, this PR and previous couple), probably worth moving this code into a testing helper function. i think you can do this by naming a file 'helper-xxx.R' in the tests folder?